README.md

```diff
@@ -6,25 +6,34 @@
 - [Why this repo?](#why-this-repo)
 - [What counts as an acceptable entry?](#what-counts-as-an-acceptable-entry)
 - [Language Specific Resources](#language-specific-resources)
-  - [Rust](#rust)
-  - [Go](#go)
-  - [Python](#python)
-  - [Ruby](#ruby)
-  - [OCaml](#ocaml)
-  - [Haskell](#haskell)
-  - [Erlang](#erlang)
-  - [Elixir](#elixir)
-  - [Common Lisp](#common-lisp)
-  - [Java](#java)
-  - [Scala](#scala)
-  - [D](#d)
-  - [Clojure](#clojure)
-  - [Nim](#nim)
-  - [Pony](#pony)
-  - [JavaScript](#javascript)
-  - [C++](#C++)
-  - [Dart](#dart)
-  - [General API Design and Tooling](#general-api-design-and-tooling)
+  - [ALGOL Language Family](#algol-language-family)
+    - [Dart](#dart)
+  - [C Language Family](#c-language-family)
```

Comment:
I don't see how Nim, Python, Ruby, Rust can be classified as C language family?

Nim is compiled but it is more similar to Ada rather than C

Python and Ruby are not even compiled languages, the type system is completly different

Go has a GC, C doesn't

Rust is diametrically opposite to C when it comes to memory safety